doc/modules/manifold.rst

```diff
@@ -418,67 +418,64 @@ Multi-dimensional Scaling (MDS)
-space and the similarities/dissimilarities.
+space and the input dissimilarities.
 
 .. figure:: ../auto_examples/manifold/images/sphx_glr_plot_lle_digits_010.png
    :target: ../auto_examples/manifold/plot_lle_digits.html
    :align: center
    :scale: 50
 
 
-Let :math:`S` be the similarity matrix, and :math:`X` the coordinates of the
-:math:`n` input points. Disparities :math:`\hat{d}_{ij}` are transformation of
-the similarities chosen in some optimal ways. The objective, called the
-stress, is then defined by :math:`\sum_{i < j} d_{ij}(X) - \hat{d}_{ij}(X)`
+Let :math:`\delta_{ij}` be the dissimilarity matrix between the
+:math:`n` input points (possibly arising as some pairwise distances
+:math:`d_{ij}(X)` between the coordinates :math:`X` of the input points).
+Disparities :math:`\hat{d}_{ij} = f(\delta_{ij})` are some transformation of
+the dissimilarities. The MDS objective, called the raw stress, is then
+defined by :math:`\sum_{i < j} (\hat{d}_{ij} - d_{ij}(Z))^2`,
+where :math:`d_{ij}(Z)` are the pairwise distances between the
+coordinates :math:`Z` of the embedded points.
 
 
 .. dropdown:: Metric MDS
 
-  The simplest metric :class:`MDS` model, called *absolute MDS*, disparities are defined by
-  :math:`\hat{d}_{ij} = S_{ij}`. With absolute MDS, the value :math:`S_{ij}`
-  should then correspond exactly to the distance between point :math:`i` and
-  :math:`j` in the embedding point.
-
-  Most commonly, disparities are set to :math:`\hat{d}_{ij} = b S_{ij}`.
+  In the metric :class:`MDS` model (sometimes also called *absolute MDS*),
+  disparities are simply equal to the input dissimilarities
+  :math:`\hat{d}_{ij} = \delta_{ij}`.
 
 .. dropdown:: Nonmetric MDS
 
   Non metric :class:`MDS` focuses on the ordination of the data. If
-  :math:`S_{ij} > S_{jk}`, then the embedding should enforce :math:`d_{ij} <
-  d_{jk}`. For this reason, we discuss it in terms of dissimilarities
-  (:math:`\delta_{ij}`) instead of similarities (:math:`S_{ij}`). Note that
-  dissimilarities can easily be obtained from similarities through a simple
-  transform, e.g. :math:`\delta_{ij}=c_1-c_2 S_{ij}` for some real constants
-  :math:`c_1, c_2`. A simple algorithm to enforce proper ordination is to use a
-  monotonic regression of :math:`d_{ij}` on :math:`\delta_{ij}`, yielding
-  disparities :math:`\hat{d}_{ij}` in the same order as :math:`\delta_{ij}`.
-
-  A trivial solution to this problem is to set all the points on the origin. In
-  order to avoid that, the disparities :math:`\hat{d}_{ij}` are normalized. Note
-  that since we only care about relative ordering, our objective should be
+  :math:`\delta_{ij} > \delta_{kl}`, then the embedding
+  seeks to enforce :math:`d_{ij}(Z) > d_{kl}(Z)`. A simple algorithm
+  to enforce proper ordination is to use an
+  isotonic regression of :math:`\delta_{ij}` on :math:`d_{ij}(Z)`, yielding
+  disparities :math:`\hat{d}_{ij}` on the same order as :math:`d_{ij}(Z)`.
```

Comment:
`IsotonicRegression`of `y=distances` on `X=dissimilarities`, giving the `y_pred=disparities` close to the `distance` but ordered as the `dissimilarities`.
```suggestion
  isotonic regression of :math:`d_{ij}(Z)` on :math:`\delta_{ij}`, yielding
  disparities :math:`\hat{d}_{ij}` on the same order as :math:`\delta_{ij}`.
```

Reply:
Hmm. No. It is correct that `IsotonicRegression` is run with `X=dissimilarities, y=distances`, but here `dissimilarities` are the input data to MDS, i.e. `\delta_ij`, and `distances` are distances in the embedding, i.e. `d_ij(Z)`. So I think it's correct to say that it is regressing "deltas on d(Z)" and that it yields disparities on the same order as "d(Z)".

Reply:
Hmm, wait, I think I take that back.

Reply:
Okay, sorry, I got confused about "regression on smth" terminology. Apparently, in regression we say that "y is regressed on X" (https://stats.stackexchange.com/questions/207425/why-do-we-say-the-outcome-variable-is-regressed-on-the-predictors). So, it should be "regression of d(Z) on deltas". Fine. I'm now trying to understand what's the correct formulation for the second part.

Reply:
Oh that might be a grammar issue then ?
I would say: isotonic regression of `y` on `X`,  yielding estimate `y_pred` (close to `y`) ordered as `X`.
You would say the other way around ?

Reply:
Okay, turns out I was confused about what "on the same order as" is supposed to mean :-) I thought it's about "order of magnitude" whereas you are clearly right -- it's about the "ordering"!

Reply:
I merged your suggestion and did another reformulation to clarify that!